libs/logs.js

```diff
@@ -0,0 +1,36 @@
+import winston from 'winston';
+
+const logger = winston.createLogger({
+  level: process.env.LOG_LEVEL || 'info',
+  format: winston.format.json(),
+  transports: [new winston.transports.Console()],
+  exitOnError: false,
+});
+
+export const log = (level, message, requestId, errorCode, customData = {}) => {
```

Comment:
I like this idea, however that would make some parameters optional. Maybe at least provide message as well, so `log(level, message, options)`? Also maybe name it `extraParams` since I think `options` implies that it's being used to modify the code behavior 🙂.

Reply:
I guess the second parameter is up for discussion in regards of the name, in my opinon neither ´options´ or ´extraParams´ are good variabel names for these extra params. We should have a name that indicates the purpose of the parameters so that it's clear for the person reading the code (y)

Reply:
I kind of want the requestId required.
Concurrent requests to lambdas makes it hard to follow logs without the requestId as a filter, then I rather make the message optional.
Is it more "messy" if I validate the object input?